src/ui/src/ambientTypes.ts

```diff
@@ -7,4 +7,10 @@ declare module "vue" {
 	}
 }
 
+declare global {
+	interface Window {
+		__WRITER_APP_CONFIG__?: WriterAppConfig;
```

Comment:
`window.__WRITER_APP_CONFIG__` is a base64 string on prod (https://app.writer.com/env/config.js).

```suggestion
		__WRITER_APP_CONFIG__?: WriterAppConfig | string;
```